### PR TITLE
Fixed telegram bot answering to replies and pin/join/leave events that don't contain any commands

### PR DESCRIPTION
**Pull request by claudiofreda**
For context: https://meta.discourse.org/t/set-up-telegram-integration-using-the-discourse-chat-integration-plugin/66603/67

### CODE REVIEW

**Review by davidtaylorhq — APPROVED** [review]

**Comment by davidtaylorhq** [comment]
Thanks for submitting this @claudiofreda! I just enabled GitHub actions for this PR, and it looks like there is a Rubocop failure. Please can you correct that. 

Also, if you're able to add an RSpec test which demonstrates the fix, that would be great!

**Comment by g5pw** [comment]
Hello! So, what can be done to merge this? Having the telegram bot respond to join/part/pin is getting annoying :/

File: lib/discourse_chat_integration/provider/telegram/telegram_command_controller.rb [hunk 1]
```diff
@@ -62,10 +64,15 @@ def process_command(message)
 
       channel = DiscourseChatIntegration::Channel.with_provider(provider).with_data_value('chat_id', chat_id).first
 
-      text_key = "unknown_chat" if channel.nil?
-      # If slash commands disabled, send a generic message
-      text_key = "known_chat" if !SiteSetting.chat_integration_telegram_enable_slash_commands
-      text_key = "help" if message['text'].blank?
+      text_key = if channel.nil?
+        "unknown_chat"
+      elsif !SiteSetting.chat_integration_telegram_enable_slash_commands or !message['text'].start_with?('/')
```
Comment (davidtaylorhq) [inline lib/discourse_chat_integration/provider/telegram/telegram_command_controller.rb:69]: Rubocop violation - please replace `or` with `||`

File: lib/discourse_chat_integration/provider/telegram/telegram_command_controller.rb [hunk 2]
```diff
@@ -62,10 +64,15 @@ def process_command(message)
...
+        "silent"
+      else
```
Comment (davidtaylorhq) [inline lib/discourse_chat_integration/provider/telegram/telegram_command_controller.rb:71]: Rubocop violation - please remove the trailing space on this line

**Comment by g5pw** [comment]
Opened #94 with applied fixes!

**Pull request closed without merging**